packages/salesforcedx-vscode-core/src/orgBrowser/browser.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ExtensionContext, TreeView, window } from 'vscode';
+import { nls } from '../messages';
+import { BrowserNode, MetadataOutlineProvider } from '../orgBrowser';
+import { telemetryService } from '../telemetry';
+import { OrgAuthInfo } from '../util';
+
+export class OrgBrowser {
```

Comment:
Needed to access the treeView and dataProvider outside of the `index.ts` file, so I created a separate container class to be an entry point for all Org Browser activity.